Organization/Organization/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,29 @@
+#import "CreateEmployeeViewController.h"
+#import "Employee.h"
+
+
+@interface CreateEmployeeViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *firstNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *salaryTextField;
+
+@end
+
+@implementation CreateEmployeeViewController
+
+
+- (IBAction)saveEmployee:(id)sender
+{
+    NSString *firstName = _firstNameTextField.text;
+    NSString *lastName = _lastNameTextField.text;
+    NSString *salary = _salaryTextField.text;
+    NSLog(@"%@", firstName);
+    NSLog(@"%@", lastName);
+    NSLog(@"%@", salary);
+    Employee* savedEmployee = [[Employee alloc] initWithFirstName:firstName lastNameInit:lastName initSalary:[salary integerValue]];
```

Comment:
Access properties with dot, not as a method. So replace `[salary integerValue]` with `salary.integerValue`